internal/service/project/data_source_project.go

```diff
@@ -137,12 +138,40 @@ func (d *projectDS) Schema(ctx context.Context, req datasource.SchemaRequest, re
 					},
 				},
 			},
+			"ip_addresses": schema.SingleNestedAttribute{
```

Comment:
can something be used from project_ip_access_list resource/ds or they're different concepts?